cmd/apply.go

```diff
@@ -97,6 +98,17 @@ func RunApply(options *ApplyOptions) error {
 	}
 	logger.Info("Loaded cluster: %s", cluster.Name)
 
+	if options.Set != "" {
+		sets := strings.Split(options.Set, ",")
+		for _, set := range sets {
+			parts := strings.SplitN(set, "=", 2)
+			err := swalker.Write(strings.Title(parts[0]), cluster, parts[1])
```

Comment:
This will crash if I enter flag as =  set1=1,set2=2,

Reply:
@prateekgogia will this do the job?